### PR TITLE
Upgrade ckan 2.9 (Only merge after CKAN redirection to maintenance page)

### PR DESCRIPTION
**Pull request by kentsanggds**
## What

Update config files for CKAN 2.9 and update cronjobs with new ccommands for Staging and Production stacks.

## Reference 

https://trello.com/c/DZJFi8z5/2613-create-prs-to-prep-for-29-upgrade

### CODE REVIEW

File: hieradata_aws/class/integration/ckan.yaml [hunk 1]
```diff
@@ -3,5 +3,13 @@
 govuk_solr::disable: true
 govuk_solr6::present: true
 
+govuk::apps::ckan::blanket_redirect_url: https://find-data-beta-integration.cloudapps.digital/ckan_maintenance
```
Comment (smford) [inline hieradata_aws/class/integration/ckan.yaml:6]: are you wanting a:

'govuk::apps::ckan::enabled: true'

somewhere here?

Comment (kentsanggds) [reply hieradata_aws/class/integration/ckan.yaml:6]: It's currently working, not sure if we need to explicitly declare that

Comment (smford) [reply hieradata_aws/class/integration/ckan.yaml:6]: ok cool!

**Review by smford — APPROVED** [review]
lgtm!

**Pull request merged**